From 94db7a169628ae3031defd278211b96a1bc8893f Mon Sep 17 00:00:00 2001
From: Alexandre Janniaux <ajanni@videolabs.io>
Date: Fri, 8 Jan 2021 15:33:53 +0100
Subject: [PATCH] configure.ac: call AM_GNU_GETTEXT

Fix failure when reconfiguring libcddb with:
 - autoconf (GNU Autoconf) 2.70
 - automake (GNU automake) 1.16.3
 - /usr/bin/autopoint (GNU gettext-tools) 0.21
 - libtool (GNU libtool) 2.4.6.42-b88ce-dirty

> configure.ac: error: AM_GNU_GETTEXT_VERSION or
> AM_GNU_GETTEXT_REQUIRE_VERSION is used, but not AM_GNU_GETTEXT
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 9aa4984..a4a85bb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -21,6 +21,7 @@ AC_SUBST(LIBCDDB_VERSION_NUM)
 
 dnl iconv uses gettext macros (dependency on config.rpath)
 AM_GNU_GETTEXT_VERSION(0.14.4)
+AM_GNU_GETTEXT([external], [need-formatstring-macros])
 
 dnl Process custom command-line options
 AC_ARG_ENABLE([loglevel],
-- 
2.30.0

